apps/web/components/banner.tsx

```diff
@@ -26,17 +26,47 @@ export const Banner: React.FC<PropsWithChildren<Props>> = ({
   variant,
   persistChoice,
 }) => {
-  const [visible, setVisible] = persistChoice
+  const bannerHeightRef = useRef<number>(0);
+  const bannerRef = useRef<HTMLDivElement>(null);
+
+  const [visible, setVisible] = !persistChoice
     ? useState(true)
-    : useLocalStorage(`unkey_banner_${persistChoice}`, true);
+    : useLocalStorage(`unkey_banner_${persistChoice}`, true, { initializeWithValue: false });
+
+  useResizeObserver({
+    ref: bannerRef,
+    onResize: ({ height }) => {
+      if (height !== bannerHeightRef.current) {
+        // Update Banners.
+        updateBannersBottomPosition();
+
+        // Set current banner height.
+        bannerHeightRef.current = height ?? 0;
+      }
+    },
+  });
```

Comment:
Orders banners bottom fixed position on window resize AND whenever the banner has a height change.